### PR TITLE
Tag Gadfly v0.6.1 [https://github.com/GiovineItalia/Gadfly.jl]

### PR DESCRIPTION
**Pull request by bjarthur**
1. fixed line_style
2. fix dep warns for julia 0.6
3. fix normalization of histogram
4. lots of fixes for Date aesthetics
5. simplified Geom.abline interface
6. fix jitter for categorical data
7. constructor refactoring

### CODE REVIEW

File: Gadfly/versions/0.6.1/requires [hunk 1]
```diff
@@ -0,0 +1,16 @@
+julia 0.5
+Colors 0.3.4
+Compat 0.8.5
```
Comment (tkelman) [inline Gadfly/versions/0.6.1/requires]: need minimum of 0.18 for parametric aliases

Comment (tkelman) [inline Gadfly/versions/0.6.1/requires]: need to update the metadata requires file for this

**Comment by ararslan** [comment]
I edited the Compat version in this PR to match @bjarthur's update to the package. Does 0.6.1 need to be retagged to include the Compat change, @tkelman?

**Comment by bjarthur** [comment]
what would be the consequences of simply merging this PR without the second commit that updates the Compat requirement?  Gadfly's dependency on Compose ensures the correct version of Compat will be used.  and the commit to Gadfly post tag v0.6.1 means it will be fixed eventually.

**Comment by ararslan** [comment]
> what would be the consequences of simply merging this PR without the second commit that updates the Compat requirement?

Probably nothing

**Comment by tkelman** [comment]
Tightening requirements needs to be done in place, a future tag with tighter requirements can't fix the existence of a past tag with requirements that are too loose. It's not guaranteed that all future versions of Compose will have a sufficiently high minimum Compat requirement (they may at some point not depend on Compat at all).

**Comment by ararslan** [comment]
Okay, so @bjarthur should delete the v0.6.1 tag and retag it?

**Comment by tkelman** [comment]
not necessarily, having the commit here is equivalent since metadata's require file takes priority for tagged versions of registered packages

**Comment by ararslan** [comment]
Okay, so this is fine as-is?

**Comment by tkelman** [comment]
This caused a BoundsError in Mads.jl https://github.com/JuliaCI/pkg.julialang.org/blob/ec1913d13f8bff8600af1fcef7431cb022e947eb/logs/Mads_0.5.log
cc @montyvesselinov

**Comment by montyvesselinov** [comment]
Thank you very much for checking in all of these. I also noticed this
failure.

-monty

**Comment by montyvesselinov** [comment]
Is this going to be fixed soon?

**Comment by tkelman** [comment]
You should open an issue on Gadfly with a small self contained test case

**Pull request merged**